resources/asciidoctor/lib/chunker/breadcrumbs.rb

```diff
@@ -15,12 +15,22 @@ def generate_breadcrumbs(doc, section)
       result = ['<div class="breadcrumbs">']
       result += generate_breadcrumb_links(section, chev).reverse
       result << '</div>'
+
       if result[2].to_s.include? 'APM'
         result[2] = chev + generate_apm_breadcrumbs(doc)
       end
       if result[2].to_s.include? 'ECS Logging'
         result[2] = chev + generate_ecslogging_breadcrumbs(doc)
       end
+      if result[2].to_s.include? 'Enterprise Search'
+        result[2] = chev + generate_enterprise_search_breadcrumbs(doc)
+      end
+      if result[2].to_s.include? 'App Search'
+        result[2] = chev + generate_app_search_breadcrumbs(doc)
+      end
+      if result[2].to_s.include? 'Workplace Search'
+        result[2] = chev + generate_workplace_search_breadcrumbs(doc)
+      end
```

Comment:
Yay! 🎉 

It looks like all of the dropdowns are the same. So I don't think we need three different `generate_*` functions. We can have three different `if` statements — one for each Search book, but then they could all call the same `generate_` function. That would reduce the duplication lower in this file.

Reply:
It would also be useful to add a term for the client books. As-is, none of these dropdowns would appear if a user is looking at a client book.

Reply:
@colleenmcginnis might even have a way to use regex to match all of the Search books and client books with one regular expression.

Reply:
Thanks @bmorelli25! I omitted the clients from my draft because (I think) I'd need to build _all_ the docs locally to test 'em. I had my hands full just getting this to a semi-working state already 😄.

Reply:
I'll have a gander at this again next week.

Reply:
Happy to help or 🍐 at some point next week if you want!